hackinsta.py

```diff
@@ -67,22 +67,22 @@ def Login(username,password):
 
 
 
-def follow(sess, username):
-	username = userExists(username)
+def follow(sess, nurrhome):
+	nurrhome = userExists(nurrhome)
 	if (username == False):
 		return	
 	else:
-		userID = username['id']
+		userID = nurrhome['id']
 		followReq = sess.post('https://www.instagram.com/web/friendships/%s/follow/' % userID)
 		print (followReq.text)
 
 
-username = str(input('Please enter a username: '))
-username = userExists(username)
-if (username == False):
+nurrhome = str(input('Please enter a username: '))
+nurrhome = userExists(nurrhome)
+if (nurrhome == False):
 	exit()

```

Comment:
hi there, i think somethink is wrong with this line:
after enterin real username: error occurs

User not found
Traceback (most recent call last):

  File "***\instahack-master\instahack-1-patch-2\hackinsta.py", line 83, in <module>
    exit()

NameError: name 'exit' is not defined